### DISCUSSION TITLE
Work with yubikey

### DISCUSSION THREAD

**Discussion by valsha** [JTD420/laravel-pgp#3] [Q&A]
Good day, great project.
Is it possible to work with Yubikey (PGP keys stored in yubikey)?
Thank you.

**Comment by JTD420** [comment]
> Good day, great project. Is it possible to work with Yubikey (PGP keys stored in yubikey)? Thank you.

Hi,

Thanks for your comment and for your interest in my Laravel package.

Regarding your question about working with Yubikey, my package should be able to work with Yubikey. Here are the instructions to follow:

- To import your PGP keys into Yubikey, please follow the instructions in this link: https://developers.yubico.com/PGP/Importing_keys.html.
- To use the `generate_keypair` method provided by my package, please refer to the wiki or readme for more information.

Unfortunately, I do not currently own a Yubikey myself, so I am not able to personally test it. However, I am in the process of getting one, and will be able to look into it more closely soon!

Thanks again for your support and let me know if you have any other questions or concerns. :)

Best regards,
JTD

**Reply by valsha** [reply]
> > Good day, great project. Is it possible to work with Yubikey (PGP keys stored in yubikey)? Thank you.
> 
> Hi,
> 
> Thanks for your comment and for your interest in my Laravel package.
> 
> Regarding your question about working with Yubikey, my package should be able to work with Yubikey. Here are the instructions to follow:
> 
>     * To import your PGP keys into Yubikey, please follow the instructions in this link: https://developers.yubico.com/PGP/Importing_keys.html.
> 
>     * To use the `generate_keypair` method provided by my package, please refer to the wiki or readme for more information.
> 
> 
> Unfortunately, I do not currently own a Yubikey myself, so I am not able to personally test it. However, I am in the process of getting one, and will be able to look into it more closely soon!
> 
> Thanks again for your support and let me know if you have any other questions or concerns. :)
> 
> Best regards, JTD

Excellent, I will wait for your results when you receive yours Yubikey  :) 
Have a nice day & best regards,
valsha

**Comment by valsha** [comment]
Hi, any news with Yubikey? Thanks.
